tools/roslaunch/src/roslaunch/rlutil.py

```diff
@@ -226,7 +225,7 @@ def check_roslaunch(f):
     # check for missing nodes
     for pkg, node_type in nodes:
         try:
-            if not roslaunch.resource_cache.find_node(pkg, node_type):
+            if not roslib.packages.find_node(pkg, node_type):
```

Comment:
Can we use cached resource for find_node? (not only find_node at https://github.com/ros/ros_comm/pull/682/files#diff-67998749e41c63572aa0901b6715e9a0R251)